spec/requests/registrations_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+RSpec.describe 'Registrations', type: :request do
+  describe 'POST /signup' do
+    subject(:request) { post signup_path, params: params, as: :json }
+
+    let(:response_body) { JSON.parse(response.body).deep_symbolize_keys }
```

Comment:
We have a helper for this :D
https://github.com/comarev/comarev/blob/master/spec/support/request_helpers.rb#L19